torax/torax_imastools/equilibrium.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2024 DeepMind Technologies Limited
```

Comment:
I think we can have a more concise package name here. Instead of `torax_imastools` I think simply `imas` is sufficient or `imas_tools`